src/sonic-host-services-data/debian/sonic-host-services-data.process-reboot-cause.service

```diff
@@ -1,7 +1,8 @@
 [Unit]
-Description=Reboot cause determination service
-After=rc-local.service
+Description=Retrieve the reboot cause from the history files and save them to StateDB
+Requires=database.service
```

Comment:
Shouldn't this change (`Requires=database.service`) be added to `determine-reboot-cause.service` too?

https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-host-services-data/debian/sonic-host-services-data.determine-reboot-cause.service#L3

Reply:
`determine-reboot-cause.service` Does not require the database to be ready. It is intended to run as early at boot as possible in order to capture the reboot reason. The reason we need two services is because `process-reboot-cause.service` must run after the DB is ready in order to push the data to the DB.